lldb/source/Expression/UserExpression.cpp

```diff
@@ -424,6 +425,14 @@ UserExpression::Execute(DiagnosticManager &diagnostic_manager,
                         const EvaluateExpressionOptions &options,
                         lldb::UserExpressionSP &shared_ptr_to_me,
                         lldb::ExpressionVariableSP &result_var) {
+  Debugger *debugger =
+      exe_ctx.GetTargetPtr() ? &exe_ctx.GetTargetPtr()->GetDebugger() : nullptr;
+  Progress progress("Running expression",
+                    m_options.IsForUtilityExpr()
+                        ? "(LLDB utility)"
+                        : m_expr_text.substr(0, 12) + "...",
```

Comment:
This will render ellipsis even if the expression is shorter than 12 characters. 
```suggestion
                        : m_expr_text.size() > 12 :  substr(0, 12) + "..." : m_expr_size(),
```